### ISSUE TITLE
Inconsistent bug prone timezone argument order

### ISSUE THREAD

**Issue by kyrylkov** [Hexagon/croner#73]
```
Cron('2023-01-23T00:00:00', { timezone: 'Asia/Kolkata' }, () => { console.log('Yay') });
```
is inconsistent vs default usage such as
```
const job = Cron('*/5 * * * * *', () => {
	console.log('This will run every fifth second');
});
```
`timezone` should come as 3rd argument, not 2nd to minimize bugs:
```
Cron('2023-01-23T00:00:00', () => { console.log('Yay') }, { timezone: 'Asia/Kolkata' });
```
Alternatively, if wishing to follow callback as the last argument convention, make it two arguments only (object + callback), like:
```
Cron({time: '2023-01-23T00:00:00', timezone: 'Asia/Kolkata' }, () => { console.log('Yay') });
```

**Comment by Hexagon** [comment]
Will leave a comment on this when i have more time. A proposed fix is pushed, which make second and third argument completely interchangeable.

**Comment by Hexagon** [comment]
Now released in 4.3.3
